modules/compliance-operator-configure-scanning.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance-operator/compliance-operator-rhacs.adoc
+:_module-type: PROCEDURE
+[id="compliance-operator-configure-scanning_{context}"]
+= Configuring the ScanSettingBinding object
+
+[role="_abstract"]
+Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to scan the cluster using the `cis` and `cis-node` profiles. 
+
+[NOTE]
+====
+This example uses `cis` and `cis-node` profiles, but {ocp} provides additional profiles. See "Understanding the Compliance Operator" in the "Additional resources" section for more information.
+====
+
+.Procedure
+
+Select one of the following options:
+
+* Use the CLI to create the YAML file and object. For example:
+
+.. Create a file called `sscan.yaml` using the following text:
++
+[source,yaml]
+----
+
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSettingBinding
+metadata:
+  name: cis-compliance
+profiles:
+  - name: ocp4-cis-node
+    kind: Profile
+    apiGroup: compliance.openshift.io/v1alpha1
+  - name: ocp4-cis
+    kind: Profile
+    apiGroup: compliance.openshift.io/v1alpha1
+settingsRef:
+  name: default
+  kind: ScanSetting
+  apiGroup: compliance.openshift.io/v1alpha1
+----
+.. Create the `ScanSettingBinding` object by running the following command:
++
+[source, terminal]
+----
+$ oc create -f sscan.yaml -n openshift-compliance
+----
+If successful, the following message is displayed:
++
+[source, terminal]
+----
+$ scansettingbinding.compliance.openshift.io/cis-compliance created
+----
+* Use the web console to create the object by performing the following steps:
+
+.. Change the active project to `openshift-compliance`.
+.. Click *+* to open the *Import YAML* window.
+.. Paste the YAML from the previous example and then click *Create*.
+
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/security_and_compliance/compliance-operator#understanding-compliance-operator[Understanding the Compliance Operator]
```

Comment:
are RHACS links typically hardcoded to go to the CP location? normally this would be an `include` statement, but maybe there's something about the RHACS that i'm missing 😄

Reply:
Yeah, they have to be hardcoded because they're in a different branch so an include or an xref won't work. :(

Reply:
gotcha, makes sense! thanks for clarifying